src/modules/Telemetry/Sensor/DPS310Sensor.cpp

```diff
@@ -0,0 +1,46 @@
+#include "configuration.h"
+
+#if !MESHTASTIC_EXCLUDE_ENVIRONMENTAL_SENSOR
+
+#include "../mesh/generated/meshtastic/telemetry.pb.h"
+#include "DPS310Sensor.h"
+#include "TelemetrySensor.h"
+#include <Adafruit_DPS310.h>
+#include <typeinfo>
```

Comment:
Just checking - is typeinfo used in this class?

Reply:
Nope, that was a copypasta from the BMP280 files I was using as examples :sweat_smile: fixed